src/compiler/transformers/declarations.ts

```diff
@@ -1568,6 +1571,32 @@ export function transformDeclarations(context: TransformationContext) {
             case SyntaxKind.ModuleDeclaration: {
                 needsDeclare = false;
                 const inner = input.body;
+
+                function updateModuleDeclarationAndKeyword(

```

Comment:
Can we move this out of `transformTopLevelDeclaration` and into `transformDeclarations`?